### ISSUE TITLE
Error: Failed to bind to the service.

### ISSUE THREAD

**Issue by ch3tan03** [MinaSamir11/react-native-in-app-review#116]
Unable to invoke in-app review. getting Error: Failed to bind to the service.

**Comment by MinaSamir11** [comment]
Can you please put your working environment

**Comment by ch3tan03** [comment]
getting this error for the android app.

**Comment by AndrewKirkovski** [comment]
same for `"react-native": "0.68.2"`

**Comment by elinahovakimyan** [comment]
Getting same error with `"react-native": "0.67.3"` and `"react-native-in-app-review": "^4.1.1"`

**Comment by ch3tan03** [comment]
> I figured out this is working on a real device with only @gmail.com account and it should be the only account. It is mentioned in the docs as well.

I have tried on a real device as well with a closed internal testing group. still, I'm getting the same error.

**Comment by MinaSamir11** [comment]
> > I figured out this is working on a real device with only @gmail.com account and it should be the only account. It is mentioned in the docs as well.
> 
> I have tried on a real device as well with a closed internal testing group. still, I'm getting the same error.

How do you log the error if you already followed the doc instructions!? 
logging error maybe you are still in development mode

**Comment by MinaSamir11** [comment]
did you build new release after integration our package. and test it on internal testing or production .!

**Comment by ch3tan03** [comment]
Yes of course it's a release build which I upload in a internal testing group for testing purpose. my build is not open for public access as its still in pilot testing phase.

**Comment by MinaSamir11** [comment]
Can you please show me how you log error or screenshot of it !

**Comment by ch3tan03** [comment]
Sorry for the delay. I was traveling. see the below release build adb logs

08-18 15:44:03.525 11136 11260 I PlayCore: UID: [10149]  PID: [11136] ReviewService : Initiate binding to the service.
08-18 15:44:03.526   520  3046 W ActivityManager: Unable to start service Intent { act=com.google.android.finsky.BIND_IN_APP_REVIEW_SERVICE pkg=com.android.vending } U=0: not found
08-18 15:44:03.526 11136 11260 I PlayCore: UID: [10149]  PID: [11136] ReviewService : Failed to bind to the service.

**Comment by ilyavf** [comment]
Same issue. Running in Internal Testing. React Native bare flow. Env:
```
"react-native": "0.64.3",
"react-native-in-app-review": "^4.1.1",
```

The exception is captured and reported with Sentry:
```
import InAppReview from 'react-native-in-app-review'
...
InAppReview.RequestInAppReview().then(...).catch(e => {
   // captured and reported to Sentry here.
});

Error: Failed to bind to the service.
{
  arguments: [
    {
      nativeStackAndroid: [
        {
          lineNumber: 77,
          file: 'AppReviewModule.java',
          methodName: 'lambda$show$1$AppReviewModule',
          class: 'com.ibits.react_native_in_app_review.AppReviewModule',
        },
        {
          lineNumber: 4,
          file: null,
          methodName: 'onComplete',
          class: 'com.ibits.react_native_in_app_review.-$$Lambda$AppReviewModule$hECUHwdnIN2K-eObeUQcvk07zHA',
        },
        { lineNumber: 23, file: null, methodName: 'run', class: 'com.google.android.play.core.tasks.a' },
        { lineNumber: 938, file: 'Handler.java', methodName: 'handleCallback', class: 'android.os.Handler' },
        { lineNumber: 99, file: 'Handler.java', methodName: 'dispatchMessage', class: 'android.os.Handler' },
        { lineNumber: 226, file: 'Looper.java', methodName: 'loopOnce', class: 'android.os.Looper' },
        { lineNumber: 313, file: 'Looper.java', methodName: 'loop', class: 'android.os.Looper' },
        { lineNumber: 8669, file: 'ActivityThread.java', methodName: 'main', class: 'android.app.ActivityThread' },
        { lineNumber: -2, file: 'Method.java', methodName: 'invoke', class: 'java.lang.reflect.Method' },
        {
          lineNumber: 571,
          file: 'RuntimeInit.java',
          methodName: 'run',
          class: 'com.android.internal.os.RuntimeInit$MethodAndArgsCaller',
        },
        { lineNumber: 1135, file: 'ZygoteInit.java', methodName: 'main', class: 'com.android.internal.os.ZygoteInit' },
      ],
      userInfo: null,
      message: 'Failed to bind to the service.',
      code: '23',
    },
  ],
  logger: console,
}
```

**Comment by AviKokare** [comment]
You can test this feature on a real device, with a Gmail account. Working good !

**Comment by AviKokare** [comment]
U can make an APK of ur app and install it on any compatible physical device. There some policies associated with the app review, please go through it.

**Comment by jenipharachel** [comment]
@AviKokare I tried that and used a proper gmail email ID to log into the app. But I was not able to see it. I was under the impression that I would be able to see it in development mode.

**Comment by AviKokare** [comment]
If your, Quotas are exceeded then please try with another Gmail account.

**Comment by jenipharachel** [comment]
Thank you! It didn't work with an APK but worked with internal beta

**Comment by AviKokare** [comment]
@jenipharachel  If it is working, then rating pop will appear once. If you want to reopen that popup then, delete the ratings from the playstore, clear the cache of playstore from the mobile setting -> apps -> manage app/application -> play store and clear the cache. U can do this process multiple time for the testing.

**Comment by jenipharachel** [comment]
@AviKokare Thank you so much! I'll keep that in mind during testing

**Comment by samimzafar** [comment]
replace emulator with physical, i mean run app to install via physical device instead of emulator. then android will works expectedly. and generate apk from react native and install it on physical device. 
but your device must have gmail account to test your app for review

**Comment by ThoaiBuiMCV** [comment]
I has create patch package for that, 
[react-native-in-app-review+4.3.3.patch](https://github.com/MinaSamir11/react-native-in-app-review/files/14507752/react-native-in-app-review%2B4.3.3.patch)
